apps/deploy-web/src/components/shared/ManagedEscrowFaqModal.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+import React from "react";
+import { FormattedNumber } from "react-intl";
+import { Alert, Popup } from "@akashnetwork/ui/components";
+import { ArrowRight } from "iconoir-react";
+
+import { useChainParam } from "@src/context/ChainParamProvider";
+import { useTotalWalletBalance } from "@src/hooks/useWalletBalance";
+
+export type Props = {
+  onClose: () => void;
+};
+
+export const ManagedEscrowFaqModal: React.FunctionComponent<Props> = ({ onClose }) => {
```

Comment:
this totally works. I just would like to point out that using the content and same popup container (portal pattern) in the root of the website is a generally better approach in terms of re-usablility. If we do all the modals like this it would have several downsides:

- it's more boilerplate code, you need to re-declare popup everywhere, imagine to refactor such thing if needed
- your can't make a component to self manage it's state (using it in a tooltip is a great example), meaning more boilerplate code on the client code